docs/SIL.rst

```diff
@@ -1767,13 +1767,19 @@ This lattice is applied to SIL by requiring well formed SIL to:
    model is strict, we do not allow for values to have unknown ownership.
 
 2. Define a map from each operand of a SILInstruction, `i`, to a constant
-   Ownership Kind, Boolean pair called the operand's `Ownership
-   Constraint`_. The Ownership Kind element of the `Ownership Constraint`_
-   determines semantically which ownership kind's the operand's value can take
-   on. The Boolean value is used to know if an operand will end the lifetime of
-   the incoming value when checking dataflow rules. The dataflow rules that each
-   `Value Ownership Kind`_ obeys is documented for each `Value Ownership Kind`_
-   in its detailed description below.
+   `Operand Ownership Semantics Kind`__ that classifies the "high level"
+   ownership semantics of the operand. Each of these high level semantics can be
+   mapped to a Ownership Kind, Boolean pair called the operand's
+   `Ownership Constraint`__. The Ownership Kind element of the
+   `Ownership Constraint`__ determines semantically which ownership kind's the
+   operand's value can take on. The Boolean value is used to know if an operand
+   will end the lifetime of the incoming value when checking dataflow rules. The
+   dataflow rules that each `Value Ownership Kind`_ obeys is documented for each
+   `Value Ownership Kind`_ in its detailed description below.
+
+.. __: Operand Ownership Semantic Kinds and Ownership Constraints
+.. __: Operand Ownership Semantic Kinds and Ownership Constraints
```

Comment:
NOTE: This is just a way in RST to enable Operand Ownership Semantics Kind, Ownership Constraint x2 in the above paragraph map to "Operand Ownership Semantic Kinds and Ownership Constraint". Evidently this is the way to do <a href="#anchor>arbitrary txt</a> in rst.